src/components/CldImage/CldImage.vue

```diff
@@ -1,13 +1,19 @@
 <template>
-  <div class="cld-image">
-    <img v-bind="imageAttrs" :style="style"/>
+  <div class="cld-image" :style="style">
+    <img v-bind="imageAttrs" :style="style" />
```

Comment:
Do we still need this?

Reply:
What do you mean by "this"?

Reply:
...and in any case I've added that cause resizing was built around having a single node - just img. Now both have to adapt to parent node size, so both have the size adjusting styles.